app/src/main/java/com/example/inventory/ui/item/ItemEntryScreen.kt

```diff
@@ -91,18 +93,21 @@ fun ItemEntryBody(
     modifier: Modifier = Modifier
 ) {
     Column(
-        modifier = modifier
-            .fillMaxWidth()
-            .padding(16.dp),
-        verticalArrangement = Arrangement.spacedBy(32.dp)
+        modifier = modifier.padding(dimensionResource(id = R.dimen.padding_medium)),
+        verticalArrangement = Arrangement.spacedBy(dimensionResource(id = R.dimen.padding_large))
     ) {
-        ItemInputForm(itemDetails = itemUiState.itemDetails, onValueChange = onItemValueChange)
+        ItemInputForm(
+            itemDetails = itemUiState.itemDetails,
+            onValueChange = onItemValueChange,
+            modifier = Modifier.fillMaxWidth()
+        )
         Button(
             onClick = onSaveClick,
             enabled = itemUiState.isEntryValid,
+            shape = MaterialTheme.shapes.small,
             modifier = Modifier.fillMaxWidth()
         ) {
-            Text(stringResource(R.string.save_action))
+            Text(text = stringResource(R.string.save_action))
```

Comment:
I don't think we need text = here

Reply:
We usually add it, is this ok to leave it as is?

Reply:
in other parts of the code its not added

Reply:
+1 to Meghan
We never add obvious single parameter names